src/TesApi.Tests/DeleteCompletedBatchJobsHostedServiceTests.cs

```diff
@@ -1,115 +1,103 @@
-﻿using System;

```

Comment:
Line endings were made consistent with the other files, in addition to the similar systemic changes made to this file as the rest of `TesApi.Tests`